src/component/Restaurants.js

```diff
@@ -42,10 +64,18 @@ function Restaurants (props) {
         );
     }
 
+    let restaurantList = null;
+    if (restaurants[region] && restaurants[region][city]) {
+        restaurantList = restaurants[region][city].map(x => {
+            const { address, images, name, slug } = x;
```

Comment:
Why not put the destructuring in the function param?

Reply:
I never really internalized destructuring in the params. But I'm doing that now. Changed